tutorials/HTML/Getting_Started/README.md

```diff
@@ -1,20 +1,16 @@
-# Getting Started With Compose for Web
-
-In this tutorial, we will create a simple web UI application using the Compose UI framework.
+# Getting Started With Compose HTML Library
 
+In this tutorial, we will create a simple web UI application using the Compose HTML Library.
 
 ## Prerequisites
 
 You need to have the following software installed before you begin:
 * JDK 11 or later
 * IntelliJ IDEA Community Edition or Ultimate Edition 2020.2 or later (you can use other editors, but for this tutorial we assume you are using IntelliJ IDEA)
 
-
 ## Creating a new project
 
-You can create the project by [downloading the template here](https://github.com/JetBrains/compose-multiplatform/tree/master/templates/web-template) or using the Project Wizard in IDEA:
-
-<img alt="Create new project" src="wizard.png" height="500" />
+You can create the project by [downloading the template here](https://github.com/JetBrains/compose-multiplatform-html-library-template).
```

Comment:
I suggest to keep https://github.com/JetBrains/compose-multiplatform-web-template for now, and rename it to `compose-multiplatform-html-library-template`

Reply:
I don't mind for now.

Reply:
Renamed the repo